tests/migration/ember-addon/steps/update-addon-package-json/public-assets.test.js

```diff
@@ -9,7 +9,7 @@ import { updateAddonPackageJson } from '../../../../../src/migration/ember-addon
 import {
   codemodOptions,
   options,
-} from '../../../../helpers/shared-test-setups/javascript.js';
+} from '../../../../helpers/shared-test-setups/typescript.js';
```

Comment:
There was an inconsistency in the test setup for the `public-assets` case. In other test files, I had imported `codemodOptions` and `options` from `shared-test-setups/typescript.js`.